src/Widget/Attribute/FieldAttributes.php

```diff
@@ -225,7 +225,7 @@ private function addValidatorAttributeHtml(
      *
      * @return string
      */
-    private function getId(): string
+    private function generateId(): string
     {
         $new = clone $this;
```

Comment:
This method doesn't mutate the state. Nothing is written to `$this` so clone isn't needed.

Reply:
done.